qiskit/quantum_info/operators/measures.py

```diff
@@ -350,6 +350,89 @@ def cvx_bmat(mat_r, mat_i):
     return sol
 
 
+def unitary_diamond_distance(channel1: Operator, channel2: Operator) -> float:
+    r"""Return the diamond distance between two unitary operators.
+
+    This function computes the completely-bounded trace-norm (often
+    referred to as the diamond-norm) of the difference of two unitary channels
+    (or unitary operators). This is often used as a distance metric in quantum
+    information This method is a more specialised implementaion of
+    :meth:`~qiskit.quantum_info.diamond_norm`.
+
+    Args:
+        channel1 (Operator): a unitary operator.
+        channel2 (Operator): a unitary operator.
+
+    Returns:
+        float: The completely-bounded trace norm :math:`\|U - V\|_{\diamond}`.
+
+    Raises:
+        QiskitError: if the input operators do not have the same dimensions or aren't unitary.
+
+    Additional Information:
+        The implementation uses an unproved result in Aharonov et al. (1998). Geometrically,
```

Comment:
You proved it though, right? If your thesis will be published soon, perhaps we can wait and cite it here? Also fine with adding it later.

Reply:
I think it's fine as is, but feel free to submit a PR to cite your work once it's published.

Reply:
Unfortunately I don't think it will be published super soon (maybe on arxiv at some point). The thesis is submitted for an examination at Oxford. I can send you the proof directly, but it shouldn't be circulated for the time being.  If it does get published, I'll make sure to open a new PR, it really shouldn't be too difficult.

Reply:
Ok sounds good.